skbase/_base.py

```diff
@@ -427,7 +427,7 @@ def get_test_params(cls, parameter_set="default"):
         """
         # if non-default parameters are required, but none have been found, raise error
         if hasattr(cls, "_required_parameters"):
-            required_parameters = getattr(cls, "required_parameters", [])
+            required_parameters = getattr(cls, "_required_parameters", [])
```

Comment:
I thought we removed the `_required_parameters` attribute?

At least `sktime` does not have it anymore, since required parameters are precisely thouse without a default in the init, and there are no kwargs.

Reply:
This existed in the code at the time we ported it over. I think we should update the logic here and in other places to be in sync with sktime as part of this release. Ideally, we'd do that after we merge this PR so we have broader suite of tests in place.

Reply:
agreed on the point of keeping scope well-defined.